cmd/dev/snapshots.cpp

```diff
@@ -227,44 +228,48 @@ void decode_segment(const SnapSettings& settings, int repetitions) {
     SILK_INFO << "Decode snapshot elapsed: " << duration_as<std::chrono::milliseconds>(elapsed) << " msec";
 }
 
+static std::unique_ptr<SnapshotBundleFactory> bundle_factory() {
+    return std::make_unique<silkworm::db::SnapshotBundleFactoryImpl>();
+}
```

Comment:
I suggest moving this factory function into `snapshot_bundle_factory_impl.hpp`, so that we can use it everywhere and have a one-liner creation for `SnapshotRepository` as below:
```
SnapshotRepository snapshot_repo{settings, bundle_factory()};
```